src/main/java/com/olgaboiar/mint/Request.java

```diff
@@ -2,15 +2,15 @@
 
 public class Request {
     private String method;
-    private String requestedFile;
+    private String uri;
     private String protocol;
 
     public String getMethod() {
         return method;
     }
 
-    public String getRequestedFile() {
-        return requestedFile;
+    public String getUri() {
+        return uri;
```

Comment:
Instead of using string for URLs, maybe we can look into using the built-in URL type.

Reply:
used in https://github.com/olgaboiar/MintHTTPServer/pull/9/commits/266dc830df2bd7ac57d2c1a4b3203403f7aa7e06